index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Fontawesome icons -->
+    <script src="https://kit.fontawesome.com/dd0d05bba7.js" crossorigin="anonymous"></script>
+
+    <!-- My stylesheet -->
+    <link rel="stylesheet" href="/css/main.css">
+    <title>Pastel UI</title>
+</head>
+<body>
+    <!-- Header starts -->
+    <header class="header">
+        <div class="header-brand"><a href="/index.html">Pastel UI</a> </div>
+
+        <button class="navbar-open-btn">
+            <i class="fas fa-2x fa-bars"></i>
+        </button>
+        <nav class="navbar hide">
+        <ul class="nav-links">
+            <li class="nav-item"><a href="/index.html">Home</a></li>
+            <li class="nav-item"><a href="">Docs</a></li>
+            <li class="nav-item"><a href="https://github.com/RakshaPawar108/Pastel-UI"><i class="fab fa-github"></i>Contribute</a></li>
+        </ul>
+        <button class="navbar-close-btn">
+            <i class="far fa-2x fa-times-circle"></i>
+        </button>
+        </nav>
+
+
+    </header>
+    <!-- Header ends -->
+
+    <!-- Hero begins -->
+    <div class="hero">
+        <div class="hero-img">
+            <img src="/images/hero-image.png" alt="hero-image">
+        </div>
+        <div class="hero-content">
+            <h1>Minimalistic, Responsive, Pastel Perfect!</h1>
+            <p>Build sophisticated and minimalistic user interfaces with easy-to-use components that are fully responsive and make your websites look Pastel-Perfect!</p>
+        </div>
+        <div class="hero-buttons">
+            <button class="btn get-started-btn"><a href="./index.html">Get Started</a></button>
+            <button class="btn github-btn"><a href="./index.html"><i class="fab fa-github"></i
+            >Github</a></button>
+        </div>
+    </div>
+    <!-- Hero ends -->
+
+    <!-- Footer starts -->
+    <footer class="footer">
+        <div class="credits"> <i class="fas fa-code"></i> with  <i class="fas fa-heart"></i>  by Raksha Pawar</div>
+
+        <div class="social-links">
+            <a href="https://twitter.com/rakshapawar108"
+              ><i class="fab fa-twitter" title="Twitter"></i
+            ></a>
+            <a href="https://github.com/RakshaPawar108"
+              ><i class="fab fa-github" title="Github"></i
+            ></a>
+            <a href="https://www.linkedin.com/in/rakshapawar/"
+              ><i class="fab fa-linkedin" title="LinkedIn"></i
+            ></a>
```

Comment:
**Fix:** Code formatting looks disturbed here! It's because of Prettier I guess.